src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const lines = table.getElementsByTagName('tr');
+const tableBody = table.querySelector('tbody');
+const counter = lines[0].getElementsByTagName('td');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+appendColumn.addEventListener('click', () => {
+  if (counter.length < 10) {
+    for (const elem of lines) {
+      const newCell = document.createElement('td');
+
+      elem.appendChild(newCell);
+    }
+  }
+});
+
+removeColumn.addEventListener('click', () => {
+  if (counter.length > 2) {
+    for (const elem of lines) {
+      elem.lastElementChild.remove();
+    }
+  }
+});
+
+appendRow.addEventListener('click', () => {
+  if (lines.length < 10) {
```

Comment:
The condition `lines.length < 10` is correct for limiting the number of rows, but ensure that this logic aligns with any specific requirements or constraints for your application.